game-headless/build.gradle

```diff
@@ -47,3 +47,7 @@ task copyShadow(dependsOn: shadowJar) {
         }
     }
 }
+
+shadowJar {
```

Comment:
Is it possible to define this in the top level build.gradle for all projects and have this be default?
I presume this is the config that is suppressing the '-all'. IMO it's a good thing to suppress, the '-all' does not add clarity considering our jar files are essentially distributables.

Reply:
I'd assume we can do a `tasks.withType(Jar)` or something like that?
IIRC that the shadowJar task is just a subtype of the default jar task, but I don't know what it's called.
Also obviously not all projects use the shadow plugin, so we might not even be able to reference the subtype.
I'll have to do some experimenting